File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> hs = new HashSet<>();

        for (int i : nums) {
            hs.add(i);
        }

        int soln = 0;
        for (Integer i : hs) {
            int curr = i;
            int count = 0;
            if (hs.contains(i - 1)) continue;
            while (hs.contains(curr)) {
                count++;
                curr++;
            }
            soln = Math.max(count, soln);
        }
        return soln;
    }
}

File: Data Structures & Algorithms/merge-intervals/submission-0.py
class Solution:
    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
        intervals.sort()

        curr = intervals[0]
        res = []

        for i in intervals:
            if not curr:
                curr = i
                continue
            if i[0] <= curr[1]:
                curr[1] = max(i[1], curr[1])
            else:
                res.append(curr)
                curr = i
        
        if curr: res.append(curr)

        return res